src/app/rundown/components/rundown-header-context-menu/rundown-header-context-menu.component.scss

```diff
@@ -0,0 +1,38 @@
+.c-context-menu {
+  display: inline-flex;
+  flex-direction: column;
+  min-width: 180px;
+  max-width: 280px;
+  background-color: rgb(255, 255, 255);
+}
+
+.c-context-menu-header {
+  padding: 10px;
+  color: #49c0fb;
+  background: #3e4041;
+  text-align: center;
+}
+
+.c-context-menu-item {
+  display: flex;
+  flex: 1;
+  flex-direction: row;
+  min-width: 64px;
+  align-items: center;
+  border: none;
+  line-height: 36px;
+  padding: 0 16px;
+
+  font-size: 1rem;
+
+  user-select: none;
```

Comment:
`user-select` can be moved to a `:host` selector.

Reply:
It has been moved.